overlay/releases.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
We shouldn't have this file, as we have overlay/releases/app-autoscaler.yml

Reply:
don't shoot the messenger... https://github.com/genesis-community/cf-app-autoscaler-genesis-kit/blob/master/devtools/pull-upstream#L59-L62